# Scale Correction (less meas_corrected = (1+S_i)measurement)
S_xg: 0.0
S_yg: 0.0
S_zg: 0.0
S_xa: 0.0
S_ya: 0.0
S_za: 0.0

# Fixed Bias Corrections
Bf_xg: 0.0
Bf_yg: 0.0
Bf_zg: 0.0
Bf_xa: 0.0
Bf_ya: 0.0
Bf_za: 0.0

# Cross Coupling Matrices
M_xyg: 0.0
M_xzg: 0.0
M_yxg: 0.0
M_yzg: 0.0
M_zxg: 0.0
M_zyg: 0.0
M_xya: 0.0
M_xza: 0.0
M_yxa: 0.0
M_yza: 0.0
M_zxa: 0.0
M_zya: 0.0

# Note: Factors like g-sensitive bias, anisoelastic error, and 
# vibro-pendulous error are ignored in this model